wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java

```diff
@@ -134,7 +134,7 @@ static void convertTrtLLM(ModelInfo<?, ?> info) throws IOException {
     static boolean needConvertOnnx(ModelInfo<?, ?> info) {
         String prefix = info.prop.getProperty("option.modelName", info.modelDir.toFile().getName());
         // modelDir could be file:///model.onnx
-        return !Files.isRegularFile(info.modelDir)
+        return !Files.isDirectory(info.modelDir)
```

Comment:
this is wrong, this causes a regression in 0.29.0.

Reply:
Changed.